### PR TITLE
BZ-1395986  Explicitly identify when logging stopped reading old logs

### PR DESCRIPTION
**Pull request by jcantrill**
To fix:

https://bugzilla.redhat.com/show_bug.cgi?id=1395986

### CODE REVIEW

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -734,6 +734,14 @@ logs from the beginning of the journal.
 
 [NOTE]
 ====
+As of {product-title} 3.3, Flentd no longer reads historical log files when using the
```
Comment (ewolinetz) [inline install_config/aggregate_logging.adoc]: s/Flentd/Fluentd

**Comment by ewolinetz** [comment]
@ahardin-rh @richm PTAL

**Comment by ahardin-rh** [comment]
LGTM. Thanks! I will just make a minor style edit in a follow-up PR.

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/aggregate_logging.adoc#install-config-aggregate-logging[Aggregating Container Logs]
|Added a NOTE indicating that, as of {product-title} 3.3, Fluentd no longer reads historical log files when using the JSON file log driver.
%

**Pull request merged**